pkg/cfn/components/composer.go

```diff
@@ -1630,3 +1632,34 @@ func NewS3BucketComposer(bucketName, repo, env string) *S3BucketComposer {
 		Environment: env,
 	}
 }
+
+// ECRRepositoryComposer contains state required for creating a ECR repository
+type ECRRepositoryComposer struct {
+	ImageName   string
+	Repository  string
+	Environment string
+}
+
+// ResourceRepositoryNameOutput returns the name of the resource
+func (e *ECRRepositoryComposer) ResourceRepositoryNameOutput() string {
+	return fmt.Sprintf("%s%s%sECRepository", e.Repository, e.Environment, e.ImageName)
```

Comment:
ECRepository -> ECRRepository

Reply:
Since both you and @paulbes agreed on this, I'll change it. But you guys do realize that results in "Elastic Container Registry Repository" right?

Reply:
But isn't that what it is called? The AWS [user guide](https://docs.aws.amazon.com/AmazonECR/latest/userguide/Repositories.html) title is "Amazon ECR private repositories".